libs/ButeMgr/butemgr.h

```diff
@@ -689,7 +690,7 @@ inline bool CButeMgr::Parse(void* pData, unsigned long size, const char* cryptKe
 		return false;
 	char* buf1 = (char*)pData;
 #if _MSC_VER >= 1300 && !defined(__clang__)
-	std::streamsize len = size;
+	int len = size;
```

Comment:
please use an unsigned integer type for size

Reply:
here you're truncating from a 32bit to an effectively 31bit value (on win32/win64)